libs/SmartStore/SmartStoreTests/SFSmartStoreTests.m

```diff
@@ -866,38 +866,6 @@ - (void)testQuerySpecPageSize
     XCTAssertEqual(querySpecPageSize, expectedPageSize, @"Page size value should reflect input value.");
 }
 
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wnonnull"
-
-- (void)testCursorTotalPages
```

Comment:
totalEntries is no longer passed as a parameter to the SFStoreCursor init method.
Instead SFStoreCursor's runs the query to figure out the number of entries. So to keep this test, we would have to actually populate the db and pass a query that returns the desired number of entries. Seems a lot just to test the following lines of code:
```objective-c
float totalPagesFloat = totalEntries / (float) querySpec.pageSize;
float totalPagesFloat = totalEntries / (float)querySpec.pageSize;	        
NSUInteger totalPages = ceilf(totalPagesFloat);
int totalPages = ceilf(totalPagesFloat);	
if (0 == totalEntries)
    totalPages = 0;
```